sdk/python/runtime/main.go

```diff
@@ -228,6 +229,13 @@ func (m *PythonSdk) WithBase() (*PythonSdk, error) {
 		WithEnvVariable("UV_NATIVE_TLS", "1").
 		WithWorkdir(path.Join(m.SourcePath, m.Discovery.SubPath))
 
+	if m.IsUvIndexUrlSpecified() {
+		m.Container = m.Container.WithEnvVariable("UV_INDEX_URL", m.IndexUrl())
+	}
+	if m.IsUvExtraIndexUrlSpecified() {
+		m.Container = m.Container.WithEnvVariable("UV_EXTRA_INDEX_URL", m.ExtraIndexUrl())
+	}
```

Comment:
The extra "...Specified" functions aren't necessary, just check for an empty string:
```suggestion
	if m.IndexUrl() != "" {
		m.Container = m.Container.WithEnvVariable("UV_INDEX_URL", m.IndexUrl())
	}
	if m.ExtraIndexUrl() != "" {
		m.Container = m.Container.WithEnvVariable("UV_EXTRA_INDEX_URL", m.ExtraIndexUrl())
	}
```